adafruit_ads1x15/ads1x15.py

```diff
@@ -170,8 +174,12 @@ def _read(self, pin):
         self._write_register(_ADS1X15_POINTER_CONFIG, config)
 
         if self.mode == Mode.SINGLE:
+            # poll conversion complete status bit
```

Comment:
I would throw these comments up in the function definition comments to describe how this works in the context of the whole function.